regen-network/regen-web
# fix: first project card spacing, title and icon

## Description

Closes: regen-network/regen-registry#1628



---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test

1. Go to https://deploy-preview-1991--regen-registry.netlify.app/ecocredits/projects and login using an address that is a credit class issuer but doesn't have any project yet
2. You should see the "create project" card with the fixes

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
